src/app/services/push.service.ts

```diff
@@ -60,65 +51,30 @@ export class PushService {
     }
   }
 
-  private initPush() {
-    PushService.pushObject = this.push.init({
-      android: {},
-      ios: {
-        alert: true,
-        badge: true,
```

Comment:
How we handle those options now?

Reply:
For now the developer can't change this. It's inside of the SDK. See the discussion on https://github.com/aerogear/aerogear-js-sdk/pull/387

Should we provides all [phonegap-push-plugin options](https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/API.md#parameters)?

Reply:
Yes. That was one of the the reasons I was skeptical to have this change in js-sdk. I think we are ok with that since we all know that target will be to have single cordova plugin

Reply:
Going to merge it so and we can introduce whatever we need in the future since it will be only a new optional parameter in the registration method